src/main/java/core/basesyntax/dao/FileReaderImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.dao;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+
+    @Override
+    public List<String> readFile(String fileName) {
+        List<String> lines = new ArrayList<>();
+        try (BufferedReader br = new BufferedReader(new java.io.FileReader(fileName))) {
+            String line;
+            br.readLine();
+            while ((line = br.readLine()) != null) {
+                lines.add(line);
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Error while reading file " + fileName, e);
+        }
+        return lines;
+    }
+}
```

Comment:
According to the checklist, classes should not begin with an empty line. Please remove the empty line at the start of the class implementation.